rfcs/text/0004-session.md

```diff
@@ -88,62 +38,85 @@ Stage 3: Add or update all remaining field definitions. The list should now be e
  - Administrative sessions on infrastructure devices
  - Administrative sessions on cloud or application management portals
- - Applications sessions (e.g. sql server odbc session, application access session)
+ - Applications sessions (e.g. sql server odbc session, web cookie based session, application access session)
+ - Cloud API access sessions
 
 
 ## Source data
 Source data expectations include:
  - Wireless Lan Controllers
- - Security appliances
+ - Security appliances (e.g. fw, waf)
  - Network admission control devices
- - Radius / tacacs servers
- - Application server logs
+ - Radius / tacacs servers (802.1x EAP/PEAP aaa)
+ - Application server logs (FTP, MySQL)
+ - Web Server, WAF, or ADC logs (USer or cookie based web ession control)
+ - APM telemetry
 
-Example 1: Meraki 802.1x Logs (WLC)
-* EAP session start)
-    * `<134>1 1580551704.928047208 my_AP events type=8021x_eap_success radio='1' vap='2' client_mac='12:34:56:78:9A:BC' client_ip='192.168.1.100' identity='JohnDoe' aid='1687088497’
+### Example 1: Meraki 802.1x Logs (WLC)
+
+EAP session start
+
+`<134>1 1580551704.928047208 my_AP events type=8021x_eap_success radio='1' vap='2' client_mac='12:34:56:78:9A:BC' client_ip='192.168.1.100' identity='JohnDoe' aid='1687088497'`
 
 802.1x EAP De-association Message
-* EAP session end
-    * `<134>1 1580551705.928047208 my_AP events type=8021x_deauth radio='1' vap='2' identity='JohnDoe' aid='1687088497’'
 
-* Note, while there is an association id (session.id) created prior to wpa/802.1x authentication, building the session event from the eap success message allows for easier integration of fields like username, client.ip, etc. in an 802.1x or WPA environment
+EAP session end
+
+`<134>1 1580551705.928047208 my_AP events type=8021x_deauth radio='1' vap='2' identity='JohnDoe' aid='1687088497'`
+
+Note, while there is an association id (session.id) created prior to wpa/802.1x authentication, building the session event from the eap success message allows for easier integration of fields like username, client.ip, etc. in an 802.1x or WPA environment.
+
+Base 802.11 Association:  (802.11 session start)
+
+`<134>1 1380653443.857790533 MR18 events type=association radio='1' vap='1' channel='2' rssi='23' aid='1687088497'`
+
+Base 802.11 Deassociation Message  (802.11 session end)
+
+`1380653443.857790533 my_AP events type=disassociation radio='1' vap='2' channel='6' reason='8' instigator='2' duration='11979.728000' auth_neg_dur='1380653443.85779053324000' last_auth_ago='5.074000' is_wpa='1' full_conn='1.597000' ip_resp='1.597000' ip_src='192.168.111.251' arp_resp='1.265000' arp_src='192.168.111.251' dns_server='192.168.111.1' dns_req_rtt='1380653443.85779053335000' dns_resp='1.316000' aid='1813578850'`
+
 
-    * Base 802.11 Association:  (802.11 session start)
-        * `<134>1 1380653443.857790533 MR18 events type=association radio='1' vap='1' channel='2' rssi='23' aid='1687088497’
+### Example 2: ASA Admin Login
```

Comment:
Perhaps we could mention "ssh" in the ASA section title as well, to make it stand out for folks looking for an ssh example?

Reply:
will do